filter2.js

```diff
@@ -4,3 +4,9 @@ var misc = [3, "true", {a: 1, b: 2}, 7, [1, 2, 3], ['a', 'b', 'c'], "my favorite
 // Don't worry about strings in the nested array (misc[5] is considered an array, not a string)
 
 // Your code goes here
+const filterStrings = function (item) {
+    return typeof item === "string"
+    }
+    var stringsOnly = misc.filter(filterStrings)
+    console.log(stringsOnly)
+    
```

Comment:
careful with your indentation! this looks a little odd, see if you can fix it on your own.

besides that, great job setting up your `filter()`! the only thing is the instructions ask you to filter the strings **out**, not keep them in. Not worried about this detail though; I feel confident you would be able to fix that without too much trouble